src/main/resources/org/jenkinsci/plugins/vSphereCloudSlaveTemplate/help-instancesMin.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    Number of instances to be provisioned beforehand.<br>
```

Comment:
Can we have this of the form:

one line summary
<p>
Multiple sentences explaining what this does.
Possibly using <br> where appropriate.

If appreciate it if you can expand the description.
Tell folks what a value of 0 does.
Tell folks what a value of 1 does, and what happens when that one agent stays busy for ages.

... and my *guess* is that if you set it to the same value as the max instances then it'll try to keep that number provisioned regardless of Jenkins workload, replacing any VMs that close e.g. due to "run once" retention strategies... but if that's the case, please state this.
(Nobody ever complains about having too much help text, but too little often results in folks raising an issue asking WTF stuff does, and I'd prefer to avoid that)